src/pybind/mgr/dashboard/frontend/src/app/shared/components/date-time-picker/date-time-picker.component.html

```diff
@@ -1,16 +1,18 @@
-<div cdsCol
-     class="form-item">
+<div cdsCol>
   <div cdsRow>
-<cds-date-picker [label]="name"
+    <cds-text-label>{{name}}  
+<cds-date-picker 
                  i18n-label
-                 placeholder="NOT PROTECTED"
+                 [placeholder]="placeHolder"
                  formControlname="expiresAt"
                  dateFormat="Y/m/d"
                  [value]="date"
                  (valueChange)="onModelChange($event)"
                  [helperText]="helperText"
                  [disabled]="disabled"
                  cdsTheme="theme"></cds-date-picker>
+    </cds-text-label>
+    <cds-text-label>
```

Comment:
show this label only if we need to show the time selector. And also remove the label from the `cds-timepicker` and add it here.
```suggestion
    <cds-text-label *ngIf="hasTime">Select a time
```